bedrock/firefox/templates/firefox/new/compare/scene1-edge-1.html

```diff
@@ -14,7 +14,7 @@
 
 {% block page_title %}{{ _('Firefox vs. Edge') }}{% endblock %}
 {% block hero_title %}{{ _('Firefox vs. Edge') }}{% endblock %}
-{% block hero_desc %}{{ _('Edge came with your PC, but Firefox gives you more. Simply put, Firefox beats Edge when it comes to privacy, portability and utility.') }}{% endblock %}
+{% block hero_desc %}{{ _('Edge came with your PC, but Firefox doesn’t hog your data. Edge runs on Google’s Chromium where data means profits. Firefox beats Edge when it comes to privacy, portability and utility.') }}{% endblock %}
```

Comment:
Has this been through legal review? I don't think we can make this claim at this point in time. MS hasn't actually started shipping a Chromium-based Edge and as far as I know haven't announced a real date, just "later in 2019."

Reply:
In the issue for this PR I asked these same questions. I agree though that this doesn't sit comfortably, so let's wait on a second round of legal review.